app/assets/stylesheets/cart.css.scss

```diff
@@ -67,10 +68,10 @@
 }
 
 #cart-total {
-    left: 350px;
-    text-align: center;
-   border: none;
-  border-radius: 5px;
+  left: 350px;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, left, text-align